instrumentation/rxjava-2.0/javaagent/rxjava-2.0-javaagent.gradle

```diff
@@ -16,4 +15,5 @@ dependencies {
   implementation project(":instrumentation:rxjava-2.0:library")
 
   testImplementation project(':instrumentation:rxjava-2.0:testing')
+  latestDepTestLibrary group: 'io.reactivex.rxjava2', name: 'rxjava', version: "(,2.2.0)"
```

Comment:
Should we limit muzzle to 2.2.0 too?

Reply:
then we would lose the `assertInverse` above